tests/array/arrays.php

```diff
@@ -391,3 +391,20 @@ function () {
             return "other-value";
         }
 ];
+
+$singleLine = [ 'do' => [ 'collapse' => 'this' ]];
+
+$multiline = [
+  'do' => [
+    'not' => 'collapse'
+  ]
+];
+
+$collapse = [
+  "not",
+  "associative"
+];
+
+$mixed = [
+  'only' => [ 'partly' => 'collapsed' ]
+];
```

Comment:
Can we add tests like:
```php
array($var
);
```
```php
array(
$var);
```

Reply:
@evilebottnawi Added some more tests!

Reply:
@czosel don't see :smile:

Reply:
Oops, I pushed to the wrong remote :sweat_smile: